console/src/it/java/org/georchestra/console/integration/RolesIT.java

```diff
@@ -0,0 +1,174 @@
+package org.georchestra.console.integration;
+
+import static org.junit.Assert.assertNotNull;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.log4j.Logger;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.ldap.core.LdapTemplate;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.context.WebApplicationContext;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+/**
+ * Integration tests for {@code /private/roles} role management API.
+ */
+@RunWith(SpringRunner.class)
+@EnableWebMvc
+@ContextConfiguration(locations = { "classpath:/webmvc-config-test.xml" })
+@PropertySource("classpath:console-it.properties")
+@WebAppConfiguration
+public class RolesIT {
+	private static Logger LOGGER = Logger.getLogger(RolesIT.class);
+
+	private @Autowired LdapTemplate ldapTemplateSanityCheck;
+
+	private @Value("${ldap_port}") int ldapPort;
+
+	private @Value("${psql_port}") int psqlPort;
+
+	public @Rule TestName testName = new TestName();
+
+	RestTemplate restTemplate = new RestTemplate();
```

Comment:
What is the purpose of restTemplate ?

Reply:
None, just an oversight from a previous test. Removed it, good catch.